microlith/grafana/cmos-doc-redirect.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Redirecting, please wait...</title>
+</head>
+<body>
+You will be redirected to the requested page shortly. If nothing happens, <a id="docsLink">click here</a>.
```

Comment:
We could probably do with a bit of justification for this script in a comment or even in the description.